server/models/Collective.js

```diff
@@ -1729,6 +1729,12 @@ export default function (Sequelize, DataTypes) {
 
     await Promise.all(promises);
 
+    // Cascade host update to events
+    const events = await this.getEvents();
+    if (events?.length > 0) {
+      await Promise.all(events.map(e => e.addHost(hostCollective, creatorUser)));
```

Comment:
Do you assume that `shouldAutomaticallyApprove` will naturally be triggered? That the events will be approved similarly to the collective?

Reply:
I checked quickly and it looks right, but let's confirm with a second pair of eyes.

Reply:
Yes, I'm assuming the `creatorUser` is an admin of the parent collective.